src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -38,56 +38,67 @@ def get_section_url(addon, section, edit=False):
 
 
 @override_settings(MEDIA_ROOT=None)  # Make it overridable.
-class TestEdit(TestCase):
+class EditTest(TestCase):
```

Comment:
I thought the convention (in amo code) was to indicate it was a base class of tests by not starting the class name with 'Test' - that's why I changed the name from `TestEdit` to `EditTest`; it was already a base class.